Gemfile

```diff
@@ -43,7 +44,7 @@ group :test do
   gem "orderly"
   gem "rails-controller-testing"
   gem "rspec-activemodel-mocks"
-  gem "rspec-rails"
+  gem "selenium-webdriver"
```

Comment:
We don't want this anymore.  I switched to using cuprite as the driver because it is supposed to be faster